src/components/securities/options/steps/relevant-dates.tsx

```diff
@@ -21,11 +22,11 @@ import { useForm } from "react-hook-form";
 import { z } from "zod";
 
 const formSchema = z.object({
-  boardApprovalDate: z.string().date(),
```

Comment:
Needed to change the zod schemas form string() to date() because the component expected so. Would love to hear about that

Reply:
That should be fine, you probably can [coerce](https://zod.dev/?id=coercion-for-primitives) as well.